Automattic/sensei
# Update lessons in sample course

### Changes proposed in this Pull Request

This PR updates the following lessons in the sample course:
- Courses
- Lessons
- Quizzes
- Questions

### Testing instructions

* Import the sample course.
* Confirm the [courses](https://senseilms.com/lesson/courses/), [lessons](https://senseilms.com/lesson/lessons/), [quizzes](https://senseilms.com/lesson/quizzes/) and [questions](https://senseilms.com/lesson/questions/) lessons match what is on SenseiLMS.com.
